src/fides/api/util/consent_util.py

```diff
@@ -284,355 +248,6 @@ def get_or_create_fides_user_device_id_provided_identity(
     return identity  # type: ignore[return-value]
 
 
-def validate_notice_data_uses(
-    privacy_notices: Iterable[Union[PrivacyNoticeWithId, PrivacyNoticeCreation]],
-    db: Session,
-) -> None:
-    """
-    Ensures that all the provided `PrivacyNotice`s data has valid data uses.
-    Raises a 422 HTTP exception if an unknown data use is found on any `PrivacyNotice`
-    """
-    valid_data_uses = [data_use.fides_key for data_use in DataUse.query(db).all()]
-    try:
-        for privacy_notice in privacy_notices:
-            privacy_notice.validate_data_uses(valid_data_uses)
-    except ValueError as e:
-        raise HTTPException(HTTP_422_UNPROCESSABLE_ENTITY, detail=str(e))
-
-
-def ensure_unique_ids(
-    privacy_notices: List[PrivacyNoticeWithId],
-) -> None:
-    """
-    Ensures that all the provided `PrivacyNotice`s have unique IDs
-    Raises a 422 HTTP exception if there is more than one PrivacyNotice with the same ID
-    """
-    ids = set()
-    for privacy_notice in privacy_notices:
-        if privacy_notice.id not in ids:
-            ids.add(privacy_notice.id)
-        else:
-            raise HTTPException(
-                HTTP_422_UNPROCESSABLE_ENTITY,
-                detail=f"More than one provided PrivacyNotice with ID {privacy_notice.id}.",
-            )
-
-
-def create_privacy_notices_util(
```

Comment:
A lot of the consent util methods related to creating out of the box configuration details have moved to plus